### ISSUE TITLE
Add status check for stabilised 

### ISSUE THREAD

**Issue by rarkins** [renovatebot/renovate#495]
Applicable for major and minor upgrades only. Applicable if no further upgrades have been released for x days. Time period should be configurable.

Right now this would be pointless if equal or less than 24 hours because we will check unpublish (#494).

**Comment by edmorley** [comment]
So one consideration for the design of this - should there be a cut-off time period after which a PR is opened regardless? eg if the stabilisation time period is set to 5 days, and the dependency releases a new version every 3 days, then no PR will ever be opened otherwise.

**Comment by rarkins** [comment]
Yes, that's a consideration. I never implemented this as I wasn't sure exactly how it could be done reliably. As an example, jest v22 was released recently and is now at 22.0.3:

```
"22.0.0":"2017-12-18T11:04:01.619Z",
"22.0.1":"2017-12-18T20:30:05.633Z",
"22.0.2":"2017-12-19T13:53:42.483Z",
"22.0.3":"2017-12-19T14:59:39.422Z"
```

Unless you're *really* keen to get the latest features, I think most people would benefit from waiting until the patches stop coming 1-2 per day. But how to implement that in practice?

Also, even if we add a status check, that doesn't mean it has to block anyone's merging. Remember it would be "pending" if not stabilised or "success" if so, so at no point would someone get a "build failed" notification.
